website_breadcrumb/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Website Breadcrumbs
+===================
+
+This module allows you to have breadcrumbs in any page of your website.
+
```

Comment:
This doesn't happends, check image:

![2016-10-04_16-06-39](https://cloud.githubusercontent.com/assets/8736623/19077435/ea507e8c-8a4c-11e6-950a-e66cbcf84e62.jpg)